app/src/main/java/com/example/mycalculator/mvp/presenter/CalculatorPresenter.java

```diff
@@ -15,6 +15,26 @@ public CalculatorPresenter(CalculatorContract.CalculatorModelContract model, Cal
     @Override
     public void numberPressed(String number) {
         model.saveNumber(number);
-        view.drawNumber(model.getResult());
+        view.drawNumber(model.getPartialResult());
+
+    }
+
+    @Override
+    public void operationSymbolPressed(String toString) {
```

Comment:
put a more specific name for the param's variable

Reply:
Done. Thx